acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/signing/BlsSigningAcceptanceTest.java

```diff
@@ -120,7 +121,32 @@ public void ableToSignUsingHashicorp() throws JsonProcessingException {
       final Path keyConfigFile = testDirectory.resolve(configFilename + ".yaml");
       METADATA_FILE_HELPERS.createHashicorpYamlFileAt(
           keyConfigFile,
-          new HashicorpSigningParams(hashicorpNode, secretPath, secretName, KeyType.BLS));
+          new HashicorpSigningParams(hashicorpNode, secretPath, secretName, KeyType.BLS),
+          Optional.empty());
+
+      signAndVerifySignature(ArtifactType.BLOCK);
+    } finally {
+      hashicorpNode.shutdown();
+    }
+  }
+
+  @ParameterizedTest(name = "{index} - Using http protocol version: {0}, with TLS: {1}")
+  @CsvSource({"HTTP_1_1, true", "HTTP_2, true", "HTTP_1_1, false", "HTTP_2, false"})
```

Comment:
It appears this is doing a test for each permutation of protocol and withTLS. In what way is the assert condition changing for each of these?

I am surprised that (HTTP_2, false) is working because wasn't that the case that wasn't working for the user?

Reply:
Http2 + Hashicorp Vault (nonTLS) is supposed to work (as in our acceptance test example). The scenario that *might not* work is Web3Signer -> Hashicorp Vault Agent (nonTLS) -> Load Balancer (TLS) -> Hashicorp Vault (TLS or nonTLS). In this scenario, Http2 over h2c (plain text) may fail by the Load balancer because Hashicorp Vault Agent forward this header when it is supposed to eliminate it.

Reply:
The assertion is that we are able to connect to Hashicorp Vault with both Http1.1 and Http/2 on TLS and nonTLS.

Reply:
> Http2 + Hashicorp Vault (nonTLS) is supposed to work (as in our acceptance test example).
> The assertion is that we are able to connect to Hashicorp Vault with both Http1.1 and Http/2 on TLS and nonTLS.

This is why I'm confused - I was expecting HTTP_2 with nonTLS to not be able to connect...am I looking at the wrong test? This test isn't related to Hashicorp?

Reply:
discussed with @usmansaleem - user's issue involved a proxy in front of the vault which this AT isn't attempting to test.